vm_core.h

```diff
@@ -595,7 +595,7 @@ typedef struct rb_vm_struct {
     unsigned int running: 1;
     unsigned int thread_abort_on_exception: 1;
     unsigned int thread_report_on_exception: 1;
-    unsigned int safe_level_: 1;
+    unsigned int ignore_deadlock: 1;
```

Comment:
It is only a suggestion. how about to name it `thread_ignore_deadlock` to show it is related to threads. Of course, "deadlock" is related to threading, so "thread_" prefix is verbose.

Reply:
Given the naming of the previous two fields, I also agree with this.